From a07542d55a42f1e0b219c0255039542a980ebac0 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Thu, 08 Jun 2023 17:54:14 +0300
Subject: [PATCH] [SW-115519] fit: dsv: add (no-)load-only flag

The DSV image should be loaded as load-only

Add the support for the "load-only" / "no-load-only" flag.
It is a FIT's implicit flag, hint the FWL, to operate the FIT, as if
it does when the "-l" option is passed as a command line argument.

Change-Id: Icea7bbea038edfda17c8f0bbdae3c1a6c059d9d8
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/board/habanalabs/common/dsv.its b/board/habanalabs/common/dsv.its
index a7a6207..f9f3e59 100644
--- a/board/habanalabs/common/dsv.its
+++ b/board/habanalabs/common/dsv.its
@@ -46,6 +46,7 @@
 			loadables = "dsv", "uboot";
 			load-bootloader;
 			load-type = <0>; /* FIT_LOAD_TYPE_RUN */
+			load-only;
 		};
 	};
 };
